src/components/IconedImage/IconedImage.tsx

```diff
@@ -0,0 +1,35 @@
+import { Image } from 'antd'
+import { ReactNode } from 'react'
+
+/**
+ * Shows an {@link Image} with a small icon in the top left.
+ */
+export const IconedImage = ({
+  url,
+  width,
+  onClick,
+  icon,
+}: {
+  url: string
+  width: number
+  icon: ReactNode
+  onClick: VoidFunction
+}) => {
+  return (
+    <div style={{ position: 'relative' }}>
+      <Image key={url} width={width} src={url} />
+      <div
```

Comment:
I think we should:
- put this on the right-hand side
- make it a bit bigger
- potentially change color to make it more salient.

I think we could either:

- use antd `Badge` component https://ant.design/components/badge/
- Use similar design to the image uploader. 

<img width="171" alt="Screen Shot 2022-07-06 at 11 32 41 PM" src="https://user-images.githubusercontent.com/94939382/177568624-6dafaf76-5981-44f1-954a-58d79c0b6673.png">

Reply:
I think badge is the wrong component here - looks like its used for notifications rather than a close button. I also tried out the blue, but it feels a bit distracting to the form. wdyt about this
![image](https://user-images.githubusercontent.com/104132113/177666374-d7e72faf-2142-4b99-a716-8604777665bb.png)

Its not exactly the same as image uploader, but I think thats ok too

Reply:
Looks perfect! Great call.